torch/_torch_docs.py

```diff
@@ -2740,7 +2740,7 @@ def merge_dicts(*dicts):
 add_docstr(
     torch.clamp,
     r"""
-clamp(input, min=None, max=None, *, out=None) -> Tensor
+clamp(input: Tensor, min: Optional[Union[Tensor, Number, _complex]] = None, max: Optional[Union[Tensor, Number, _complex]] = None, *, out: Optional[Tensor] = None) -> Tensor # noqa: B950
```

Comment:
this noqa doesn't do what you think it does

Reply:
:thinking: Thought it would help to suppress warning like this

```
>>> Lint for torch/nn/functional.py:

  Advice (FLAKE8) B950
    line too long (134 > 120 characters)
    See https://github.com/PyCQA/flake8-bugbear#list-of-warnings

        4034  |    reduce: Optional[bool] = None,
        4035  |    reduction: str = "mean",
        4036  |) -> Tensor:  # noqa: D400,D402
    >>> 4037  |    r"""multilabel_soft_margin_loss(input: Tensor, target: Tensor, weight: Optional[Tensor] = None, reduction: str = 'mean') -> Tensor
        4038  |
        4039  |    See :class:`~torch.nn.MultiLabelSoftMarginLoss` for details.
        4040  |    """

```